### DISCUSSION TITLE
How to escape curly braces?

### DISCUSSION THREAD

**Discussion by kirschkern** [kazupon/vue-i18n#1545] [Q&A]
In my localized string I have double curly braces like this:

`"defaultMailContent" : "Hello {{USER_NAME}},\n\n\nRegards",`

When running this, the parser throws an error in the console:

```
Message compilation error: Not allowed nest placeholder
1  |  Hello {{USER_NAME}},
```

I've tried to escape the {{ using a backslash "\\" but this doesn't change anything.

How can I use {{ and }} within my string and tell the parser to not translate it?

**Comment by faaktap** [comment]
<div v-pre>{{ data }}</div>

**Reply by kirschkern** [reply]
I don't understand your answer. May you give a full example?
